jobclient/project.clj

```diff
@@ -1,15 +0,0 @@
-;; This project file exists to make it easier to develop the the job client 
```

Comment:
💯

Reply:
Yeah, that comment didn't make much sense to me either...